### ISSUE TITLE
Mouse/keyboard navigation does not work

### ISSUE THREAD

**Issue by aroman** [robcowie/SublimeTODO#24]
What it says on the tin.

Doesn't work at all for me, no errors logged in the console.

**Comment by robcowie** [comment]
Not good. Ok, to get the obvious out of the way, could you remove the package and install it (using package control). There have been reports of failed updates so you might be using an old version.

Also. What platform are you on?, what version of ST2?, and what version of SublimeTODO is reported by Package Control?

**Comment by aroman** [comment]
Thanks for the quick reply.

I'm on Ubuntu 11.04 with ST2 2181 using Vintage mode.

Package Control reports SublimeTODO v2012.03.29.17.22.07.

I uninstalled and reinstalled the package to no avail. Is there a way of changing the alt+double click modifier key? I have a feeling that might be conflicting with a desktop-level keybinding. Regardless, it's not just the mouse that isn't working -- it's the actual keys to navigate the results.

**Comment by robcowie** [comment]
Yup, it's vintage mode. With my setup, keyboard nav is borked with vintage mode on but alt-double-click works. No doubt your hunch about OS-level keybindings conflicting with alt-click is accurate. So... I can make the double-click modifier key customisable without much effort. I'm not yet sure what I can do about vintage mode. I'll look into it.

**Comment by taboada** [comment]
Hello, got any follow up about this ? I'm using Vintage mode too, and cannot figure out how I can make TODO react to the standard "n", "p" and "return" keys. The alt-dblclik works though.
Thank you for any help you can provide …

**Comment by robcowie** [comment]
To be honest, I haven't looked at it for a while. I'll try to set some time aside this week to look at it.

I think the best solution I can provide is to move the move the key bindings into the User folder (a pattern followed by most plugins now) so that they override the defaults provided by other plugins (including Vintage). I'll also add a scope to the keybinding context so they operate only in TODO results.

Vintage mode users are then free to re-define the shortcuts if they wish, and the defaults (n/p) will only clash with vim-mode in the results document.

**Comment by taboada** [comment]
Very cool ! Looking forward testing that soon (and I'll be happing to test and report back to you if it helps ;-)

Thank you very much.

**Comment by merlinthered** [comment]
Hi and thanks for this very useful package. If you could also restrict the mouse click command to only work in the TODO results that would be awesome. Avoiding conflicts with other stuff using the same binding in a different context is always a good thing.
